stringbuilder.go

```diff
@@ -56,6 +59,24 @@ func (s *StringBuilder) AppendRune(char rune) *StringBuilder {
 	return s
 }
 
+// Appends a single integer to the StringBuilder instance
+func (s *StringBuilder) AppendInt(integer int) *StringBuilder {
+	return s.Append(strconv.Itoa(integer))
+}
+
+// Appends a single boolean to the StringBuilder instance
+func (s *StringBuilder) AppendBoolean(flag bool) *StringBuilder {
```

Comment:
```suggestion
func (s *StringBuilder) AppendBool(flag bool) *StringBuilder {
```

This would be the more idiomatic naming in golang

Reply:
Done